Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
I may kick myself for asking this, but since you already merged in the change, too late!  :)  I got a RepRapDiscount Full Graphic Controller and noticed the default behaviour of the knob is opposite of what I have.  Turn it counter-clockwise and the menu selection moves down.  Is the standard going to be the way I have it on mine?  Personally, turning it counter-clockwise seems backwards to me no matter where the knob is located.